src/client/user/UserHero.js

```diff
@@ -58,6 +66,16 @@ const UserHero = ({
                 hasCover={hasCover}
                 isFollowing={isFollowing}
                 onTransferClick={onTransferClick}
+                isActive={() =>
```

Comment:
Quite a lot of code there. It probably could be extracted to variable (if using boolean is possible) or method.